gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/MysqlMultiActiveLeaseArbiter.java

```diff
@@ -242,7 +242,6 @@ private void runRetentionOnArbitrationTable() {
     ScheduledThreadPoolExecutor executor = new ScheduledThreadPoolExecutor(1);
     Runnable retentionTask = () -> {
       try {
-        Thread.sleep(10000);
```

Comment:
not concerned... more wondering... did this become redundant, given the scheduled TP executor?

Reply:
Yes this became redundant and I think in merging or perhaps in the code change this line was missed in removing.